phases/is_public.go

```diff
@@ -1,9 +1,55 @@
 package phases
 
-import "fmt"
+import (
+	"bufio"
+	"os"
+	"strconv"
+	"strings"
 
-// IsPublic ...
+	"github.com/bitrise-io/go-utils/log"
+)
+
+type visibilityOption struct{
```

Comment:
You are using this structure to have two fields, one of the fields at least is unique. This type of structure matches to the built-in map type. Use map instead. Also the code base not using any benefits of having a struct.